unabridged-works-volume-2/original/17-queries-romans-14-20.adoc

```diff
@@ -27,7 +27,7 @@ he, by the finger of his pure power, toucheth some of the hearts of the children
 and in the pure demonstration of his living Spirit is found working there.
 Religion, true religion, is not a matter of opinion (as men,
 who have not tasted of the power, are apt to imagine),
-but a new creation and work of God in the heart of the creature,
+but a__ new creation and work of God__ in the heart of the creature,
```

Comment:
something is funky about this line, I think the underscores after `a__` probably should be moved over next to `new`, (and can be single underscore then,) like:

```ASCIIDOC
but a _new creation and work of God_ in the heart of the creature,
```